ci/init-plugin-test-service.sh

```diff
@@ -41,3 +41,20 @@ docker exec -i rmqnamesrv /home/rocketmq/rocketmq-4.6.0/bin/mqadmin updateTopic
 
 # prepare vault kv engine
 docker exec -i vault sh -c "VAULT_TOKEN='root' VAULT_ADDR='http://0.0.0.0:8200' vault secrets enable -path=kv -version=1 kv"
+
+# prepare openfunction env
+wget https://github.com/buildpacks/pack/releases/download/v0.27.0/pack-v0.27.0-linux.tgz
+tar -zxvf pack-v0.27.0-linux.tgz
+
+# please fork and repalce this(./samples/functions/knative/hello-world-go) if you want to update function
+git clone https://github.com/OpenFunction/samples.git
+git clone https://github.com/jackkkkklee/my-samples.git
```

Comment:
@tzssangglass @spacewander I think we may avoid these dependencies from personal repos or this repo will be messy.

Reply:
How about create dir like ./t/plugin/openfunction/test-uri , ./t/plugin/openfunction/test-body ,./t/plugin/openfunction/test-header and put functions code in there

Reply:
@jackkkkklee Good idea. I think that's OK.